src/bio/bio_xstream.c

```diff
@@ -216,6 +313,20 @@ bio_spdk_env_init(void)
 		return rc;
 	}
 
+	if (spdk_conf_find_section(NULL, "Vmd") != NULL) {
+		/**
+		 * Enumerate VMD devices and hook them into the SPDK PCI
+		 * subsystem.
+		 */
+		rc = spdk_vmd_init();
```

Comment:
And spdk_vmd_hotplug_monitor() needs be periodically called on 'init' xstream to monitor VMD hotplug event. This can be done when implementing hotplug.

Reply:
@NiuYawei yes this will be done in hotplug implementation. will add a TODO since it is apart of VMD implementation